apps/sap-commerce-cloud/src/components/Dialog/ProductList.tsx

```diff
@@ -20,7 +20,7 @@ export class ProductList extends React.Component<Props> {
 
   render() {
     const isFieldTypeSymbol =
-      get(this.props.sdk.parameters.invocation, 'fieldType', '') === 'Symbol';
+      (get(this.props.sdk.parameters.invocation, 'fieldType', '') as string) === 'Symbol';
```

Comment:
Interesting use of `as string` and `as string[]` above. Did this code throw an error without the explicit casting to string?

Reply:
yeah it had some bizarre type error in the build and a quick google provided this an answer. I think it looks kinda dumb but it's also fairly inert and seems to make typescript happy 🤷